system/atlas/values.yaml

```diff
@@ -2,6 +2,7 @@ global:
   region:
 owner-info:
   support-group: observability
+  service: prometheus
```

Comment:
Why prometheus? This service is with @stefanhipfel :)

Reply:
It's already in our support group. Since it provides a Prometheus SD it felt fitting